src/components/gui/gui.jsx

```diff
@@ -108,14 +98,18 @@ const GUIComponent = props => {
                                     />
                                 </Box>
                                 <Box className={styles.extensionButtonContainer}>
-                                    <IconButton
+                                    <button
                                         className={classNames(styles.extensionButton, {
                                             [styles.hidden]: !enableExtensions
                                         })}
-                                        img={addExtensionIcon}
-                                        title={addExtensionMessage}
+                                        title={'Add Extension'}
```

Comment:
Wouldn't we still want this localized? Is it used by screen readers, or is there another attribute that should be there? Or is the only reason for this to have a unique tag for testing?